tests/Feature/Api/V1/MobileBootstrapTest.php

```
@@ -0,0 +1,206 @@
+        ->assertJsonPath('data.this_week_days', 0)
+        ->assertJsonPath('data.activity.13', ['date' => MOBILE_BOOTSTRAP_TODAY, 'count' => 0]);
+});
+
+function createBootstrapReading(
+    User $user,
+    int $bookId,
+    string $dateRead,
+    string $createdAt,
+    int $chapter = 1,
+): ReadingLog {
+    return ReadingLog::factory()->for($user)->createOne([
```

PHPStan (2.2.7)

[error] Function createBootstrapReading() should return App\Models\ReadingLog but returns Illuminate\Database\Eloquent\Model.


(return.type)